### DISCUSSION TITLE
how to uninstall all installed packages

### DISCUSSION THREAD

**Discussion by faom94** [cryinkfly/Autodesk-Fusion-360-for-Linux#176] [Q&A] [answered]
nothing runs for me, like uninstalling all installed packages

**Comment by imWACC0** [comment]
Is there an answer? I'm trying to reinstall, and am in a loop. (err:sync:RtlpWaitForCriticalSection section 00430094 "dlls/ntdll/heap.c: main process heap section" wait timed out in thread 03f4, blocked by 0300, retrying (60 sec))

**Comment by cryinkfly** [comment, marked as answer]
Please look here: https://github.com/cryinkfly/Autodesk-Fusion-360-for-Linux/wiki/Documentation#uninstall-autodesk-fusion-360
